### DISCUSSION TITLE
Pull down to refresh scrollable list on lowest position

### DISCUSSION THREAD

**Discussion by RomanSuvorov** [gorhom/react-native-bottom-sheet#659] [Q&A]
Hello everybody,
Who can help deal with customGestureHandling func (https://github.com/gorhom/react-native-bottom-sheet/tree/master/example/src/screens/advanced/customGestureHandling)
I need to reload the data when I pull the BottomSheetSectionList down from the lowest position (on dragEnd event)
I will be very grateful for suggestions or advice

When I add onRefresh method for BottomSheetSectionList, the refresh works only in the highest position and this is not an obvious solution

**Comment by RomanSuvorov** [comment]
Any suggestions?

**Reply by i-pelypenko** [reply]
Hello, I am having the same problem. Have you found a solution? Would be very thankful for the help

**Reply by zakthedev** [reply]
I need help with this as well, did you get any ideas how to solve this? I need access to a trigger function when the bottom sheet is released, one solution I thought about is using onAnimate but it only trigger when the origin snap point and the target are different (ex. bottom sheet snapping from index 0 to 1, but it won't work from index 1 to 1)
